### DISCUSSION TITLE
How to remove certain parts...

### DISCUSSION THREAD

**Discussion by EriksonEnterprises** [spatie/laravel-permission#2035] [Q&A]
Hello. I was curious if there's a way to basically convert to Fortify for everything except for the permissions part of spatie

Unfortunately the roles are being handled by spatie as well, and I'm not familiar enough with spatie to know how to use fortify roles and replace the current spatie roles

A developer used spatie for my application which is why I'm asking

**Comment by erikn69** [comment]
>how to use fortify roles

This section is not for `fortify` discussions, but fortify doesn't use roles, it is just a frontend agnostic authentication backend implementation for Laravel. Fortify registers the routes and controllers needed to implement all of Laravel's authentication features, including login, registration, password reset, email verification, and more, 
so, what roles are you talking about?

**Reply by erikn69** [reply]
Mark this as answered

**Comment by EriksonEnterprises** [comment]
Thanks. I ended implementing just the part you mentioned instead of
everything else. There was a video I was following along that showed doing
roles but like you said it's not part of fortify so I'm good to go

Sorry for the confusion
